### ISSUE TITLE
Microbial, proteomic, and metabolomic profiling of the estrous cycle in wild house mice

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#298]
Microbial, proteomic, and metabolomic profiling of the estrous cycle in wild house mice – Tereza Matějková – Microbiology Spectrum
https://journals.asm.org/doi/10.1128/spectrum.02037-23

**Comment by omojokunoluwatomisin10** [comment]
can i be assigned to this @SvetlanaUP

**Comment by omojokunoluwatomisin10** [comment]
i am done with the curation, need a review

**Comment by SvetlanaUP** [comment]
Ready for review https://bugsigdb.org/Study_1005

**Comment by omojokunoluwatomisin10** [comment]
@SvetlanaUP please  kindly delete experiment 1 (signature 1-5)-they are relative abundance

**Comment by SvetlanaUP** [comment]
Hi @omojokunoluwatomisin10, your curated paper is with tag "needs review" that means now it's in our hands under review and you are kindly asked to not make any changes while we are reviewing it.

**Comment by omojokunoluwatomisin10** [comment]
@SvetlanaUP, DULY NOTED

**Comment by Scholarpat** [comment]
Hello @SvetlanaUP,

Please help delete signatures 3 - 7 of experiment 1 in this curation;
https://bugsigdb.org/Study_1005

Thank you very much.

**Comment by Scholarpat** [comment]
Hi @SvetlanaUP,
This curation is now complete.
Here are the changes I made to this curation:

- Study design: Changed from 'time series / longitudinal observational' to 'laboratory'.
- Data transformation: Changed from 'relative abundances' to 'centered log ratio'.
- Included significance threshold.
- Adjusted alpha diversity for both experiments.
- Marked the 'incomplete' experiment as 'complete'.
- Changed group names for experiment 2.
- Curated figures 3B and 5D as signatures for experiments 1 and 2, respectively (figures 1 and 5B were previously curated).

Thank you!
